lib/fastlane/plugin/yarn/helper/yarn_helper.rb

```diff
@@ -8,21 +8,27 @@ class YarnHelper
       attr_accessor :commands
       attr_accessor :options
       attr_accessor :package_path
+      attr_accessor :project_root
       attr_accessor :yarn
 
-      def initialize(package_path: nil)
+      def initialize(package_path: nil, project_root:nil)
         self.package_path = package_path
 
         if self.package_path
           self.yarn = "cd #{File.dirname(self.package_path)} && yarn"
+        elsif
```

Comment:
wont this `elsif` and `else` both be called if `package_path` is `nil`?

Reply:
You are right, would have catch it if I dud not have forgotten to write a real test for project_root.
Pushed the fixed inn d7ff91b :)